<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox {
	background-color:orange;
}</string>
     </property>
     <property name="title">
      <string>Menu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonContinue">
        <property name="text">
         <string>Continuer une partie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNewGame">
        <property name="text">
         <string>Nouveau jeu (local)</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/trainButton</normaloff>:/images/trainButton</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNewGameNetwork">
        <property name="text">
         <string>Nouveau jeu (r√©seau)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLoadSave">
        <property name="text">
         <string>Charger une partie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonProfil">
        <property name="text">
         <string>Profil</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOptions">
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
